ui/src/components/graph/graph.tsx

```diff
@@ -38,6 +38,10 @@ const Graph: React.FC<Props> = ({ data, nodeId }) => {
     setElements(layoutedElements);
   }, [layoutedElements, data, nodeId]);
 
+  const isFeature = (featureType: string) => {
+    return featureType === 'feathr_anchor_feature_v1' || featureType === 'feathr_derived_feature_v1'
+  }
```

Comment:
There is already one isFeature in ui code, please try to reuse that. 

Can you pls also make 'feathr_anchor_feature_v1' an enum?

Reply:
- moved 'isFeature' into utils under graph/ so that other files can access it more easily
- make all nodes types as enums in utils